websites/E/EternalNetwork TM/presence.ts

```diff
@@ -65,7 +65,7 @@ presence.on("UpdateData", async () => {
 			if (pathname.startsWith("/")) presenceData.state = ttl;
 
 			if (pathname.includes("/wp-admin")) {
-				presenceData.state = "Using administrating power over the website !";
+				presenceData.state = "Viewing an admin panel";
```

Comment:
`an` suggests there are many panels? It should be 
```suggestion
				presenceData.state = "Viewing the admin panel";
```

Reply:
Agree with you, I missed this message - fixed.

Reply:
Still unresolved

Reply:
Changes applied.